views/mixins/overviewBody.jade

```diff
@@ -0,0 +1,46 @@
+mixin renderOverviewFields(obj, hasTitle, hasSubtitle)
+	section.group
+		div.panel
+			if hasTitle
+				div.panel-heading=obj.label
+			div.panel-body
+				if hasSubtitle
+					h4=obj.label
+				-	if (obj.body)
```

Comment:
no. removed unneccesary `-` charachter. Have refactored the overview jade template.